### ISSUE TITLE
Doesn't capture cache.get_or_set

### ISSUE THREAD

**Issue by PeterJCLaw** [adamchainz/django-perf-rec#319]
I would expect that `cache.get_or_set` either be recorded as a `get` and (optionally) a `set`, or perhaps as its own entry. Instead it seems to be ignored completely:
``` python
class TestCache(TestCase):
    def test_get(self):
        with django_perf_rec.record():
            django_cache.get('foo')

    def test_set(self):
        with django_perf_rec.record():
            django_cache.set('foo', 42)

    def test_get_or_set(self):
        with django_perf_rec.record():
            django_cache.get_or_set('foo', 42)
```
This results in
``` yaml
TestCache.test_get:
- cache|get: foo
TestCache.test_get_or_set: []
TestCache.test_set:
- cache|set: foo
```
It looks like this is a result of some code which aims to identify "internal" usages of the cache, though I'm not really sure.

**Comment by adamchainz** [comment]
`get_or_set` was added after django-perf-rec was created, so it wasn't in the list of operations to record. Fixed in #320, released in version 4.10.0.

**Comment by PeterJCLaw** [comment]
Amazing, thanks for the speedy response!
